free/src/main/scala/cats/free/Free.scala

```diff
@@ -218,10 +231,17 @@ object Free extends FreeInstances {
     pure(()).flatMap(_ => value)
 
   /**
+   * a FunctionK, suitable for composition, which calls mapK
+   */
+  def mapK[F[_], G[_]](fk: FunctionK[F, G]): FunctionK[Free[F, ?], Free[G, ?]] =
```

Comment:
why `FunctionK[F, G]` here but `F ~> G` in most other places?

Reply:
This method is a copy-paste-rename from the original `Free.compile`. I'll go through and switch `FunctionK` to `~>` in the files modified by this PR, for consistency.